### PR TITLE
Fix use of THREE.ARUtils.placeObjectAtHit in spawn-at-surface example

### LINKED ISSUE #49: spawn-at-surface example don't use THREE.ARUtils.placeObjectAtHit properly
THREE.ARUtils.placeObjectAtHit(object, hit, easing=1, applyOrientation=false) is not used properly in the spawn-at-surface example and inside of the ARReticle

### PR DESCRIPTION
**Pull request by lunzhang**
closes #49

### CODE REVIEW

File: examples/spawn-at-surface.html [hunk 1]
```diff
@@ -251,8 +251,8 @@
     // the cube where the hit occurred
     THREE.ARUtils.placeObjectAtHit(cube,  // The object to place
                                    hit,   // The VRHit object to move the cube to
-                                   true, // Whether or not we also apply orientation
-                                   1);    // Easing value from 0 to 1; we want to move
+                                   1, // Easing value from 0 to 1; we want to move
+                                   true);   // Whether or not we also apply orientation
                                           // the cube directly to the hit position
```
Comment (jsantell) [inline examples/spawn-at-surface.html]: I think this line should go up with the 'Easing value...' comment

Comment (jsantell) [reply examples/spawn-at-surface.html]: Looks good with that change!

**Comment by jsantell** [comment]
could you also change this in the reticle.html example as well?

**Comment by lunzhang** [comment]
I don't see placeObjectAtHit in reticle.html.

**Comment by jsantell** [comment]
Oops, I mean [here](https://github.com/google-ar/three.ar.js/blob/c591c6a40fb7677eb4c21ade98acc4bfa061d143/src/ARReticle.js#L71) -- we can handle in a follow up as well!

**Comment by lunzhang** [comment]
What do you mean handle in a follow up?

**Comment by jsantell** [comment]
We can fix the reticle in another commit in the future I mean, I'll file another patch for that. Thanks for the fix!

**Comment by lunzhang** [comment]
Ah ok.  No Problem. :smile:

**Comment by jsantell** [comment]
If you could move that one comment line up with the other one, I'll get this landed

**Comment by lunzhang** [comment]
Oops.  Fixed Now.

**Pull request merged**